Assets/HoloToolkit/Input/Scripts/Voice/SpeechInputSource.cs

```diff
@@ -59,8 +62,8 @@ protected override void Start()
                 {
                     keywords[index] = Keywords[index].Keyword;
                 }
-
-                keywordRecognizer = new KeywordRecognizer(keywords);
+                                
```

Comment:
Nit: remove line changes.

Reply:
OK, as this is my first pull request, is this something I should fix?

Reply:
Yes, all you have to do is push the changes to your branch and they'll automatically show up here.

Reply:
@cybergoulion Just always hit ctrl+k+d in visual studio to fix formatting

Reply:
Spaces removed and recommited. I see the changes here so I guess I did it. Well it went smooth for the first time :-) 
Thanks for the help.